nums/core/array/application.py

```diff
@@ -439,6 +400,90 @@ def find_diag_output_blocks(X: BlockArray, total_elements: int):
             raise ValueError("X must have 1 or 2 axes.")
         return rarr
 
+    def _stack_copy(self, X):
```

Comment:
Can you document this function (and all functions)? Include descriptions of what the function expects as input, what it outputs, and any additional details you think are worth mentioning.